CONTRIBUTING.md

```diff
@@ -15,6 +15,58 @@ propose changes to this document in a pull request.
 * Perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Aprojectatomic%2Fatomicapp)
   to see if a similar issue has already been submitted
 
+## Developers
+
+First of all, fork the [repository](https://github.com/projectatomic/atomicapp) and clone your github repository:
+
+```shell
+$ git clone https://github.com/<your-username>/atomicapp
+```
+
+### Install this project for development
+Simply run
+
+```shell
+$ virtualenv venv
+$ . venv/bin/activate
+$ cd atomicapp/
+$ python setup.py develop
+$ pip install -r test-requirements.txt
+$ alias atomicapp=<path-to-venv>/venv/bin/atomicapp
+$ alias sudo='sudo '
```

Comment:
imo, this is too complicated and not friendly for development, we added `make install`, can't we just use that?

i'd be open to modify Makefile to add a "virtual" environment variable, etx. `make dev`. 

but I'm against this way. each time a person wants to develop they'd have to setup all these aliases again each time.